### ISSUE TITLE
Title and role attributes are missing for functional control  in Publish

### ISSUE THREAD

**Issue by ramdev1980** [percussion/percussioncms#784]
**Describe the bug**
Title and role attributes are missing for functional control  in Publish screen. 

**To Reproduce**
1. Go to Publish (http://localhost:9992/cm/app/?view=publish&site=test&path=%2FSites%2Ftest%2F&pathType=page)
2. Click on any site.
3. Mouse over on  the back button, gear icon, publish dropdown components.
4. No title/label displayed.
5. Same is true for other  visible elements on page.

**Expected behavior**
Title/Label should be displayed on each component of the screen.

**Comment by ramdev1980** [comment]
Completed

**Comment by ramdev1980** [comment]
Completed
